cmd/snap/cmd_snap_op.go

```diff
@@ -666,7 +666,13 @@ func (x *cmdRefresh) showRefreshTimes() error {
 		return err
 	}
 
-	fmt.Fprintf(Stdout, "schedule: %s\n", sysinfo.Refresh.Schedule)
+	if sysinfo.Refresh.Timer != "" {
+		fmt.Fprintf(Stdout, "timer: %s\n", sysinfo.Refresh.Timer)
+	} else if sysinfo.Refresh.Schedule != "" {
+		fmt.Fprintf(Stdout, "schedule: %s\n", sysinfo.Refresh.Schedule)
+	} else {
+		return errors.New("internal error, both refresh.timer and refresh.schedule are empty")
```

Comment:
Either this is incorrect, or the documentation on the fields above is incorrect, since they disagree. The documentation for the fields says that for both fields, the value is omitted if not in use. That seems to imply that it is possible for them both to be empty if the user hasn't set either.

As an aside, in those cases I think we use a colon after internal error ("internal error: ...").